src/HotChocolate/Filters/src/Types.Filters/FilterInputType.cs

```diff
@@ -33,7 +33,8 @@ public FilterInputType()
         {
             var descriptor = FilterInputTypeDescriptor<T>.New(
                 context.DescriptorContext,
-                typeof(T));
+                typeof(T),
+                context.DescriptorContext.GetFilterConvention());
```

Comment:
seeing this now I really do not like that we added this argument... we should use the context.